src/cookie-handler.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const bcrypt = require('bcrypt');
+const auth = require('./auth.js');
+
+var usersCollection = JSON.parse(fs.readFileSync('./data/users.json', 'utf8'));
+const cookieName = 'logged-in';
+
+//Login
+const setCookie = function (req, res) {
+    var cookie = req.cookies[cookieName];
+    if (cookie === undefined) {
```

Comment:
!cookie? also you can just rewrite the whole if the other way round, that would reduce complexity